### PR TITLE
attempt at changing algorithm part1

### PR DESCRIPTION
**Pull request by argenisRO**

### CODE REVIEW

File: scrabble/scrabble.py [hunk 1]
```diff
@@ -32,14 +32,32 @@
...
-        wordsLoaded.append(line.strip().lower())
-    print(len(wordsLoaded), "words loaded.")
-    return wordsLoaded
+    wordList = []
+    for e in inFile:
+        wordList.append(e.strip().lower())
+    inFile.close()
+
+    counter = 0
+    for letter in allWords:
```
Comment (pjrt) [inline scrabble/scrabble.py]: This isn't a letter, it is a `word`. EDIT: nvm, it is a letter? Is this the key in the dict?

Comment (argenisRO) [reply scrabble/scrabble.py]: yeah its the key

Comment (pjrt) [reply scrabble/scrabble.py]: btw, in python u can do `for key, value in dict:`. This way you don't need to do `allWords[letter]` below (you would just do `value.append...`.

Comment (argenisRO) [reply scrabble/scrabble.py]: Oh right! I completely forgot about that, I'll keep it in mind

**Pull request merged**